polaris-service/src/main/java/io/polaris/service/catalog/BasePolarisCatalog.java

```diff
@@ -704,7 +705,7 @@ public List<TableIdentifier> listViews(Namespace namespace) {
   }
 
   @Override
-  protected BasePolarisViewOperations newViewOps(TableIdentifier identifier) {
```

Comment:
This seems unrelated to the NPE fix

Reply:
Yeah, Let me remove this and fix with separate PR.